src/lpython/parser/semantics.h

```diff
@@ -720,6 +720,31 @@ static inline ast_t* BOOLOP_01(Allocator &al, Location &loc,
 #define COMPARE(x, op, y, l) make_Compare_t(p.m_a, l, \
         EXPR(x), cmpopType::op, EXPRS(A2LIST(p.m_a, y)), 1)
 
+char* unescape(Allocator &al, LCompilers::Str &s) {
+    std::string x;
+    for (size_t idx=0; idx < s.size(); idx++) {
+        if (s.p[idx] == '\\' && s.p[idx+1] == 'n') {
+            x += "\n";
+            idx++;
+        } else if (s.p[idx] == '\\' && s.p[idx+1] == '\'') {
+            x += "'";
+            idx++;
+        } else if (s.p[idx] == '\\' && s.p[idx+1] == '\"') {
+            x += '"';
+            idx++;
+        } else if (s.p[idx] == '\\' && s.p[idx+1] == ' ') {
+            x += "\\ ";
+            idx++;
+        } else if (s.p[idx] == '\\') {
+            x += "\\";
+            idx++;
```

Comment:
Comment this, check the output, uncomment or add new lines, ...; play around with this. Please make sure that you get the same output as CPython ast